library/x509_crt.c

```diff
@@ -72,6 +72,9 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <dirent.h>
+#define mbedtls_strlen     strlen
+#else
+#define mbedtls_strlen     _tcslen
```

Comment:
This seems to cause `undefined reference to '_tcslen'` whe building with MinGW-w64 (v5.0.2)